sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -320,3 +562,65 @@ def test_get_trial_metrics(katib_client, test_name, kwargs, expected_output):
     except Exception as e:
         assert type(e) is expected_output
     print("test execution complete")
+
+
+@pytest.mark.parametrize("test_name,kwargs,expected_output", test_tune_data)
+def test_tune(katib_client, test_name, kwargs, expected_output):
+    """
+    test tune function of katib client
+    """
+    print("\n\nExecuting test:", test_name)
+
+    PYTORCHJOB_KIND = "PyTorchJob"
+    JOB_PARAMETERS = {
```

Comment:
Do you need this since you just use the default values from Training SDK: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/constants/constants.py#L162?

Reply:
I added `JOB_PARAMETERS` here because it’s required in the following sections of the `tune` API:
https://github.com/kubeflow/katib/blob/b25f7ba37a60710191666b936ed47598c2de4ac2/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L637-L662
https://github.com/kubeflow/katib/blob/b25f7ba37a60710191666b936ed47598c2de4ac2/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L718-L724
I defined and patched it in the unit test to avoid importing `JOB_PARAMETERS` externally. This way, the test runs independently without needing to fetch `JOB_PARAMETERS` from outside the test environment.

Reply:
>I defined and patched it in the unit test to avoid importing JOB_PARAMETERS externally. This way, the test runs independently without needing to fetch JOB_PARAMETERS from outside the test environment.

@helenxie-bit Why we don't want to install `kubeflow-training` SDK as part of our unit tests and fetch values from there instead of re-define them in the Katib unit test ?

Reply:
Oh, I initially thought it would be better to avoid external dependencies in the unit tests, which is why I redefined those values. However, I can use this way (install `kubeflow-training` SDK as part of the unit test) if that’s preferred.